frontend/apps/example/src/app/page.tsx

```diff
@@ -74,7 +75,7 @@ export default function Home() {
 
     return (
         <div className={styles.page}>
-            <h1 style={{ marginBottom: '40px' }}>Pydantic Form</h1>
+            <h1 style={{ marginBottom: '20px' }}>Pydantic Form</h1>
```

Comment:
Maybe slowly get started with applying theme and the `css` prop over the `styles` prop. Assuming we use emotion in this repo.